src/Models/Category.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Salehhashemi\Repository\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+use Salehhashemi\Repository\Database\Factories\CategoryFactory;
+
+/**
+ * @property int $id
+ * @property string $name
+ * @property \Illuminate\Database\Eloquent\Collection|\Salehhashemi\Repository\Models\Post[] $posts
+ */
+class Category extends Model
```

Comment:
Since the models are only used for testing, we can move them to the 'tests/TestSupport/Models' directory.